### DISCUSSION TITLE
12-01

### DISCUSSION THREAD

**Discussion by ngsy** [ChristmasTest/christmas#37] [Daily_Meeting]
### 🎅🏻 회의 내용

- TMI
윤혁님 : 오늘 진짜 춥다! 아까 보니까 영하 8도다 나가면 죽는다...오늘 술마신다 왕창마신다 먹고 죽는다. 무조건 만난다.
슬희님 :  오오 이거 끝나고 2시간 이동해서 위워크 새로운 팀원을 만나러 가십니다. 페어프로그래밍을 하십니다. 술 마실예정 건들지마셈
수영님:  크리스마스 케이크 먹고싶데욧... 주문 준비 중 24년도 상남자 다죽었어!
지각자(수님): 얼렁뚱땅뚱땅 잤어요 늦잠 숙취 만취  학교 프로젝트 중이라 정신없음.. 🔥

- 어제 무엇을 하셨나요? (ex 강의 , 과제 등등)
윤혁님 : 애니메이션 넣었음 (징글벨) , 강의 다보셨고 , 과제는 next로는 다 바꿨는데 안되는 기능들이 있음.
슬희님 : 강의 다듣고 스토리북 공부하고 과제 ing 
수영님 : 강의 다봤고, 깃 날렸음 복구했음, 과제 다시 시작 , 스토리북 공부 ,
지각자(수님):  학교 다녀왔음, 강의 봤음 

-스토리북  추가 결정사항 
title은 컴포넌트 폴더 구조 그대로 가져가기

```export default {title: "Components/Button/JingleBell", component : ~~}```

### ❄️ 오늘 할일

1. 스토리북 작성하기
2. 과제하기
3. 운동하기
4. 감기 조심하기

### ⛄️ TMI

요즘 붕어빵이 제철이니 참고하세요👍

**Comment by hanseulhee** [comment]
붕어빵 제철 ...?
